### PR TITLE
Ignore first `v` in a tag if present

### PR DESCRIPTION
**Pull request by eirnym**
v is still a recommended prefix. This add a support for it

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -66,5 +66,8 @@ static def getTag() {
         else
             tagVersion = tagVersionToken[0]
     }
+    if (tagVersion.length() > 1 && tagVersion.startsWith('v')){
```
Comment (hannesa2) [inline build.gradle:69]: Isn't `tagVersion.length() > 1 &&` obsolete ?

**Comment by hannesa2** [comment]
To be honest, this introduce an unnecessary complexity on the wrong place.
If you really want to avoid this, I recommend to make a check on release pipeline, which fails, when tag isn't a number with dots.

File: build.gradle (hunk 1 above)
Comment (eirnym) [reply build.gradle:69]: GitHub still recommend to add this prefix and this code prevents to add this prefix into java version.

**Comment by eirnym** [comment]
> To be honest, this introduce an unnecessary complexity on the wrong place.
> If you really want to avoid this, I recommend to make a check on release pipeline, which fails, when tag isn't a number with dots.

makes sense. Let me think for a day or two. As most libraries and apps I know using MacPorts or software for Linux have this prefix in tag version. 

PS: Java should not have this prefix at all. and I agree with this completely

File: build.gradle (hunk 1 above)
Comment (hannesa2) [reply build.gradle:69]: Why ? What is the purpose of it ? Maybe there is a valid reason, but I can't image why (currently)

When there is a `v`, is there `w`, 'x'.... as well ?

**Comment by eirnym** [comment]
Close for now. it's not an important feature. We can return to it if we'll need it later

**Pull request closed without merging**